console/services/group_service.py

```diff
@@ -149,7 +154,7 @@ def get_app_detail(self, tenant, region_name, app_id):
         res['principal'] = app.username
         res['service_num'] = group_service_relation_repo.count_service_by_app_id(app_id)
         res['backup_num'] = backup_record_repo.count_by_app_id(app_id)
-        res['share_num'] = share_repo.count_complete_by_app_id(app_id)
+        res['share_num'] = share_repo.get_app_share_record_count_by_groupid(app_id)
```

Comment:
Only sum by the record of completed shareing?

Reply:
get_app_share_record_count_by_groupid will only count the completed shares.

```
def get_app_share_record_count_by_groupid(self, group_id):
    return ServiceShareRecord.objects.filter(group_id=group_id, step=3).count()
```

Reply:
I think you should calculate records for all states.

Reply:
Fixed